### PR TITLE
Event/Calendar model

### PR DESCRIPTION
**Pull request by szekelymarton00**
Please check thanks!
Calendar and Event model have been made together.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -30,4 +30,5 @@ dependencies {
 	runtime('mysql:mysql-connector-java')
 	compileOnly('org.projectlombok:lombok')
 	testCompile('org.springframework.boot:spring-boot-starter-test')
+	compile 'com.google.api-client:google-api-client:1.23.0'
```
Comment (adamgyulavari) [inline build.gradle]: this shouldnt be in this pr

File: src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/MainController.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package com.greenfoxacademy.opal.kalendaryo.kalendaryo;
+
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class MainController {
+
+
+
+}
```
Comment (adamgyulavari) [inline src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/MainController.java]: have a new line at every file's end

File: src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/model/Calendar.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+  @Id
+  private String id;
+  private String name;
+  private String kind;
+  private String summary;
+
+
+  @OneToMany
+  @JoinColumn(name = "id")
```
Comment (adamgyulavari) [inline src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/model/Calendar.java]: this should be probably `calendar_id` or something similar

File: src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/model/Creator.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
+package com.greenfoxacademy.opal.kalendaryo.kalendaryo.model;
+
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+
+@Entity
+public class Creator {
```
Comment (adamgyulavari) [inline src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/model/Creator.java]: this should be one model with the `Organizer`

File: src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/model/Creator.java [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+
+  @Id
+  private String id;
+  private String displayname;
+  private String email;
+
+  @OneToMany
+  List<EventModel> eventsList;
```
Comment (adamgyulavari) [inline src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/model/Creator.java]: you will have `organizedEvents` and `createdEvents`

File: src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/model/Calendar.java [hunk 6]
```diff
@@ -31,12 +31,12 @@ public Calendar() {
...
-    return id;
+  public String getCalendar_id() {
+    return calendar_id;
   }
 
-  public void setId(String id) {
-    this.id = id;
+  public void setCalendar_id(String calendar_id) {
+    this.calendar_id = calendar_id;
   }
```
Comment (Phaedra144) [inline src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/model/Calendar.java]: Please follow the styleguide, variables should be always lowerCamelCase. Please correct.

File: src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/model/EventModel.java [hunk 7]
```diff
@@ -43,12 +40,12 @@ public void setSummary(String summary) {
...
-    return id;
+  public String getEvent_id() {
+    return event_id;
   }
 
-  public void setId(String id) {
-    this.id = id;
+  public void setEvent_id(String event_id) {
+    this.event_id = event_id;
   }
```
Comment (Phaedra144) [inline src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/model/EventModel.java]: Please follow the styleguide, variables should be always lowerCamelCase. Please correct.

**Pull request closed without merging**